src/map/battle.cpp

```diff
@@ -8037,12 +8039,13 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 						skillratio += -100 + ( 570 + 20 * pc_checkskill( sd, ABC_MAGIC_SWORD_M ) ) * skill_lv + 5 * sstatus->spl;
 						RE_LVL_DMOD(100);
 						break;
-					case TR_METALIC_FURY: {
-							int area = skill_get_splash(skill_id, skill_lv);
-							int count = map_forcountinarea(skill_check_bl_sc,target->m,target->x - area,target->y - area,target->x + area,target->y + area,5,BL_MOB,SC_SOUNDBLEND);
-							skillratio += -100 + (2200 + 300 * count) * skill_lv + 5 * sstatus->spl;
-							RE_LVL_DMOD(100);
-						}
+					case TR_METALIC_FURY:
+						skillratio += -100 + 2600 * skill_lv;
+						if (tsc && tsc->getSCE(SC_SOUNDBLEND))
+							skillratio += 1000 * skill_lv;
+						if (sd)
+							skillratio += 2 * pc_checkskill(sd, TR_STAGE_MANNER) * sstatus->spl;
```

Comment:
```suggestion
						if (tsc && tsc->getSCE(SC_SOUNDBLEND)){
							skillratio += 1000 * skill_lv;
							skillratio += 2 * pc_checkskill(sd, TR_STAGE_MANNER) * sstatus->spl;
						}
```
The description (of Sigma) sounds like this should only apply to targets that are marked with SC_SOUNDBLEND.

Reply:
Before it should have been 1.5 x SPL against any target. Now I read it as 1.5 x SPL without SC_SOUNDBLEND and 2 x SPL with SC_SOUNDBLEND

Reply:
Possible, the part without SC_SOUNDBLEND is not clear.